go.mod

```diff
@@ -39,9 +39,14 @@ require go.etcd.io/bbolt v1.3.9
 
 require (
 	github.com/canonical/go-sp800.108-kdf v0.0.0-20210315104021-ead800bbf9a0 // indirect
-	github.com/canonical/tcglog-parser v0.0.0-20230929123437-16b3d8d08691 // indirect
+	github.com/canonical/tcglog-parser v0.0.0-20240820013904-60cf7cbc7c5d // indirect
+	github.com/intel-go/cpuid v0.0.0-20220614022739-219e067757cb // indirect
 	github.com/kr/pretty v0.2.2-0.20200810074440-814ac30b4b18 // indirect
 	github.com/kr/text v0.1.0 // indirect
+	golang.org/x/exp v0.0.0-20240416160154-fe59bbe5cc7f // indirect
 	golang.org/x/term v0.8.0 // indirect
 	maze.io/x/crypto v0.0.0-20190131090603-9b94c9afe066 // indirect
 )
+
+// github.com/intel-go was taken over
+replace github.com/intel-go/cpuid => github.com/aregm/cpuid v0.0.0-20220614022739-219e067757cb
```

Comment:
We need a fix in secboot for this. @chrisccoulson ?

Reply:
Yes, I'm going to push this to the @canonical org.